src/widgets.ts

```diff
@@ -125,6 +129,29 @@ export abstract class Widget<
     })
   }
 
+  handleOAuthRedirect(redirectURL: string): void {
+    try {
+      const urlStruct = new URL(redirectURL)
+      const status = urlStruct.searchParams.get("status")
+      const memberGuid = urlStruct.searchParams.get("member_guid")
+
+      const message = {
+        mx: true,
+        type: `oauthComplete/${status}`,
+        metadata: {
+          member_guid: memberGuid,
+        },
+      }
+
+      const iframeElement = this.iframe.contentWindow
+
+      iframeElement?.postMessage(message, this.targetOrigin)
+    } catch (error) {
+      console.error("Unable to postMessage OAuth URL: ", redirectURL)
+      throw error
```

Comment:
This error won't actually go anywhere since the listener is async. Not sure what throwing here gives us.

Reply:
Yeah, I kinda though the same thing really, I'll just console error the original error message so we have a complete error message and leave it at that?